lib/cli/versions.json

```diff
@@ -1,17 +1,17 @@
 {
-  "@storybook/angular": "6.0.0-beta.23",
-  "@storybook/aurelia": "6.0.0-beta.23",
-  "@storybook/ember": "6.0.0-beta.23",
-  "@storybook/html": "6.0.0-beta.23",
-  "@storybook/marionette": "6.0.0-beta.23",
-  "@storybook/marko": "6.0.0-beta.23",
-  "@storybook/mithril": "6.0.0-beta.23",
-  "@storybook/preact": "6.0.0-beta.23",
-  "@storybook/rax": "6.0.0-beta.23",
-  "@storybook/react": "6.0.0-beta.23",
-  "@storybook/riot": "6.0.0-beta.23",
-  "@storybook/server": "6.0.0-beta.23",
-  "@storybook/svelte": "6.0.0-beta.23",
-  "@storybook/vue": "6.0.0-beta.23",
-  "@storybook/web-components": "6.0.0-beta.23"
+  "@storybook/angular": "6.0.0-beta.25",
```

Comment:
@shilman I think these versions should be bumped automatically with the release no? However, I'm not sure they are used for now.